tests/usb_phy.py

```diff
@@ -155,7 +191,6 @@ def run(self):
         self.start_test()
 
         for i, event in enumerate(self._session.events):
-
             event.drive(self, self._session.bus_speed)
```

Comment:
Do we need `enumerate` here? Looks like `i` has been factored out, could just be:
```
    for event in self._sesion.bus_speed:
        event.drive(self, self._session.bus_speed)
```
Also, is `drive` ever called with a bus speed that is anything other than `self._session.bus_speed`? Could just de-reference 'self' inside the function, save the cost of argument marshalling? (Although obv. if there's any chance this needs to change, keep it as a parameter, always....)

Reply:
I guess at some point I was used for something... removed.

Reply:
In future we might want to send other values, but I'll fix it for now.

Reply:
Actually, eventually Id like to remove passing usb_phy to drive so I think I'll leave it as is for now